help/markdown/contributing.md

```diff
@@ -31,15 +31,23 @@ It turns `*.md` (Markdown with embedded code snippets) and `*.fsx` files (F# scr
 
 * Add a new git remote in order to retrieve upstream changes.
 
-        git remote add upstream https://github.com/fsharp/FAKE.git
+<pre>
+    <code class="lang-bash">
```

Comment:
This is because 
````` 
```bash
`````

is not working, correct?

Reply:
Ah okey that is due to the bug you reported in fsf

Reply:
Yes. I have to add those by hand, otherwise i get somethink like `<pre class="fsnip"><code lang="bash">` and prism.js does not like those.
The ones for the commandline output are even worse. I did not manage to get them working at all so i had to leave them as is.